.github/pr-commenter.yml

```diff
@@ -0,0 +1,67 @@
+comment:
+  on-update: recreate
+  header: |
+    Thank you for contributing to `exercism/elixir` :purple_heart: :tada:. This is an automated PR comment :robot: for the **maintainers** of this repository that helps with the PR review process. You can safely ignore it and wait for a maintainer to review your changes.
+
+    Based on the files changed in this PR, it would be good to pay attention to the following details when reviewing the PR:
+
+  footer: |
+    ---
+    Automated comment created by [PR Commenter](https://github.com/angelikatyborska/pr-commenter-action) :robot:.
+
+  snippets:
+    - id: any-file-changed
+      files:
+        - '*'
+        - '**/*'
+      body: |
+        - **General steps**
+            - :trophy: Does this PR need to receive a label with a modifier (`reputation/contributed_code/{minor,major}`)? (A medium reputation amount is awarded by default, see [docs](https://github.com/exercism/docs/blob/main/product/reputation.md#4-creating-pull-requests))
```

Comment:
```suggestion
            - :trophy: Does this PR need to receive a label with a reputation modifier (`reputation/contributed_code/{minor,major}`)? (A medium reputation amount is awarded by default, see [docs](https://github.com/exercism/docs/blob/main/product/reputation.md#4-creating-pull-requests))
```

Reply:
I'd like to link to the docs bit of the website, rather than the repo, as:
1. That's where the docs publicly live, and are long-lasting (they 301 automatically on renames for example)
2. The repo isn't "stable" in terms of it's structure

We could leave this until launch but then update afterwards?

Reply:
Yeah, sure. The snippet bodies can be updated later without any side effect if their ids stay the same.